src/erlcloud_sns.erl

```diff
@@ -387,45 +389,45 @@ list_platform_applications(NextToken, Config) ->
             Doc),
     proplists:get_value(applications, Decoded, []).
 
--spec list_platform_applications/3 :: (undefined|string(), string(), string()) -> [sns_application()].
+-spec list_platform_applications/3 :: (undefined|parameter(), parameter(), parameter()) -> [sns_application()].
```

Comment:
Line 392 is too long: -spec list_platform_applications/3 :: (undefined|parameter(), parameter(), parameter()) -> [sns_application()]..